src/_scss/pages/homepage/hero/_covid-hero.scss

```diff
@@ -63,7 +90,7 @@ $dot-color: #dcefef;
                     }
                     span {
                         width: 100%;
-                        color: $color-white;
+                        color: rgb(48, 48, 48);
```

Comment:
nit-picking: why not "#303030" like the other colors?

Reply:
updated for consistency!